stylesheets/components/image.css

```diff
@@ -0,0 +1,21 @@
+.img-responsive{
+    max-width: 100%;
+    height:auto;
+    object-fit: cover;
+}
+.img-round{
+    object-fit: cover;
+    height: 100%;
+    width: 100%;
+    border-radius: 50%;
+}
+
+.img-lg{
+    width: 7rem;
+    height:7rem;
+}
+
+.img-xlg{
+    width: 14rem;
+    height:14rem;
+}
```

Comment:
Can add extra line to avoid this. Its a good practice.

Reply:
Thanks Raghav for the review.